src/redis-check-aof.c

```diff
@@ -38,14 +38,16 @@
 #define AOF_CHECK_EMPTY 1
 #define AOF_CHECK_TRUNCATED 2
 #define AOF_CHECK_TIMESTAMP_TRUNCATED 3
+
 typedef enum {
-    RAW_AOF,
-    RDB_PREAMBLE,
-    MULTI_PART,
+    RESP_AOF,
+    RDB_PREAMBLE_AOF,
+    MULTI_PART_AOF,
```

Comment:
i'd rather these have a uniform prefix
```suggestion
    AOF_RESP,
    AOF_RDB_PREAMBLE,
    AOF_MULTI_PART,
```

Reply:
OK